### ISSUE TITLE
As I use Full screen start menu...

### ISSUE THREAD

**Issue by looknum3** [Elestriel/TaskbarTools#18]
Is there any way of the program recognizes the start screen opened state as a maximized window? 
What I mean is somehow it activate the second state and keeps the design uniform.
My maximized window state is the same transparency as the Start screen, so it is very pretty.
![max_window](https://cloud.githubusercontent.com/assets/25712388/22857403/8899a89e-f08b-11e6-8fa3-b165cb5b0769.png)
With no maximized windows, the taskbar is fully transparent, so it makes the start screen bit ugly...
![on_desktop](https://cloud.githubusercontent.com/assets/25712388/22857404/889b3fba-f08b-11e6-9baa-709e5f775d3b.png)

**Comment by DjCrqss** [comment]
Yeah, that's true, is there a way to detect when you are on the start menu to change?

**Comment by xSwagboy69x** [comment]
agreed, would be interested in getting this implemented :)

**Comment by olliecheng** [comment]
This is implemented on [TranslucentTB](https://github.com/TranslucentTB/TranslucentTB) if you want to work on it.

**Comment by NeeharVijay** [comment]
can you tell ma what the amount of opacity is in picture 1
